addons/mail/static/src/models/composer.js

```diff
@@ -140,6 +143,32 @@ registerModel({
                 composerView.update({ hasToRestoreContent: true });
             }
         },
+        async performRPClinkPreview(url) {
+            const linkPreviews = await this.messaging.rpc({
+                route: '/mail/link_preview',
+                params: { url, channel_id: this.activeThread.id },
```

Comment:
shouldn't they be allowed in chatter/thread too and not only channels?

Reply:
I don't see why we should limit them to channel only. They work in thread just fine.
I think your point is that I use `channel_id` where it should be `thread_id`.